### DISCUSSION TITLE
How ArtifactHub is managing Helm chart channels?

### DISCUSSION THREAD

**Discussion by framctr** [artifacthub/hub#4016] [Q&A] [answered]
I found that the Crossplane project is just showing its master channel chart in [ArtifactHub](https://artifacthub.io/packages/helm/crossplane/crossplane) as described in [crossplane/crossplane#5866](https://github.com/crossplane/crossplane/issues/5866). This is evident when looking at chart versions, but also when clicking on the install button it is shown only the master channel.

Is there a way to show both master and stable Helm chart channels in ArtifactHub?

PS. for clarity the 2 channels are available at `https://charts.crossplane.io/stable` and `https://charts.crossplane.io/master`

**Comment by tegioz** [comment, marked as answer]
Hi @framctr 👋

Artifact Hub doesn't really have a channels concept for Helm charts repositories. However, it's possible for an organization to publish multiple repositories.

The Crossplane organization could:

- Create a Crossplane organization in Artifact Hub and [claim the ownership](https://artifacthub.io/docs/topics/repositories/#ownership-claim) of that repository. At the moment it belongs to the Helm organization, but this is not ideal for a number of reasons (i.e. not being able to claim the verified or official status, or not receiving notifications when something is wrong with the repo).
- Add a second Helm charts repository (i.e. `crossplane-stable`), to solve the issue you describe.
- [Request the official status](https://artifacthub.io/docs/topics/repositories/#official-status) for this repository, as this helps users when deciding among multiple available options (and with search results positioning as well).

Hope this helps 🙂
